HISTORY.rst

```diff
@@ -3,6 +3,85 @@
 Release History
 ---------------
 
+
+1.2.0 (2023-03-09) Taylor
+++++++++++++++++++++++++++
+
+**Special notes**
+
+New features:
+
+* Implement radius for NetworkGrid.get_neighbors #1564
+
+Some highlights for the perf improvements:
+
+* Use getattr for attribute strings in model data collection #1590 this is a 2x speedup over the relevant line, but not sure if compared to the rest of the function
```

Comment:
We should probably either remove the "but not sure if compared to the rest of the function" part or benchmark the overall speedup.